content/docs/manuals/worker-configuration.md

```diff
@@ -0,0 +1,198 @@
+{{< /callout >}}
+
+### Configuring the NetManager
+
+You can configure the NetManager by editing the file at `/etc/netmanager/netcfg.json`. Here you can configure the following properties:
+
+| Property          | Description                                                                                                        |
+|-------------------|--------------------------------------------------------------------------------------------------------------------|
+| NodePublicAddress | The address by which the worker node can be reached                                                                |
+| NodePublicPort    | The port by which the worker node can be reached (default 50103)                                                   |
+| ClusterUrl        | The URL/address by which the cluster can be reached                                                                |
+| ClusterMqttPort   | The port by which the cluster MQTT broker can be reached (default 10000)                                           |
+| DefaultInterface  | Should the system have multiple default interfaces (e.g. `eth0` and `wlan0`) the appropriate one must be specified |
+| Debug             | Toggles more verbose logging                                                                                       |
+| MqttCert          | The path to the certificate file to facilitate [MQTTS](../networking-internals/mqtt-authentication)                |
+| MqttKey           | The path to the key file to facilitate [MQTTS](../networking-internals/mqtt-authentication)                        |
+
+### Running the NodeEngine
+
+Once the worker components have been installed the NodeEngine can be run with `sudo NodeEngine`. This will start the NodeEngine service per the default configuration.
+When running it for the first time, or changing the cluster the worker is being connected to, use:
+
+```bash
+sudo NodeEngine -a <Address of Cluster>
+```
+
+{{< callout context="note" title="Detach the NodeEngine" icon="outline/info-circle" >}}
+To run the NodeEngine in the background use the `-d` flag.
+{{< /callout >}}
+
+### Configuring the NodeEngine
+
+The NodeEngine is configured interactively with `NodeEngine config [command]`, the following config commands are available:
+
+#### addon
+
+Enables or disables [addons](../../concepts/oakestra-extensions/addons/).
+
+| Command       | Options    | Description                                                                                                        |
+|---------------|------------|--------------------------------------------------------------------------------------------------------------------|
+| `FLOps`       |`[on/off]`  | Enables the [FLOps](../../concepts/flops/overview) addon                                                           |
+| `imageBuilder`|`[on/off]`  | Enables the imageBuilder addon                                                                                     |
```

Comment:
This is a universal "addon" that allows running higher priviledged containers.
If you like you can give this example where this addon is used: https://www.oakestra.io/docs/concepts/flops/internals/image-building-process/

Reply:
I am well aware of that and I do not expect you to do so 😅
I think it is fine.